modules/react/tooltip/lib/TooltipContainer.tsx

```diff
@@ -78,6 +78,11 @@ export const TooltipContainer = styled('div')<TooltipContainerProps>(
       },
     },
 
+    '[data-popper-reference-hidden]': {
+      visibility: 'hidden',
+      pointerEvents: 'none',
+    },
+
```

Comment:
Was this needed here for any reason? It is conflicting with a recent PR to `prerelease/major`

Reply:
no, removing